### ISSUE TITLE
Lint failing due to missing mypy types

### ISSUE THREAD

**Issue by tbpg** [googleapis/docuploader#73]
```
nox > mypy docuploader
docuploader/__main__.py:24: error: Library stubs not installed for "pkg_resources" (or incompatible with Python 3.6)
docuploader/__main__.py:24: note: Hint: "python3 -m pip install types-pkg_resources"
docuploader/__main__.py:24: note: (or run "mypy --install-types" to install all missing stub packages)
docuploader/__main__.py:24: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 9 source files)
```

**Comment by tbpg** [comment]
See https://source.cloud.google.com/results/invocations/ea8521fd-882d-40ca-b222-4f45ee4fdd64/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fdocuploader%2Fcontinuous%2Fcontinuous/log.

@busunkim96 is this something we need to fix in the base image?

**Comment by parthea** [comment]
Hi @tbpg ,

I have a fix in #74. Please can you take a look?

**Comment by tbpg** [comment]
Thank you, @parthea!
